routes/web.php

```diff
@@ -31,7 +31,8 @@
         Route::get('/{user}', 'show')->name('users.show');
         Route::post('/', 'store')->name('users.store');
         Route::put('/{user}', 'update')->name('users.update');
-        Route::delete('/{user}', 'destroy')->name('users.delete');
+        Route::delete('/{user}', 'destroy')->name('users.destroy');
+        Route::get('/tasks/{user}', 'showUserTasks')->name('users.tasks');
```

Comment:
```suggestion
        Route::get('/tasks/user/{user}', 'showUserTasks')->name('users.tasks');
```
Nhìn route ai nghĩ là id của user đâu e

Reply:
nhưng mà trước /tasks thì có một cái /users rồi mà anh ?

Reply:
À oke , lỗi a k để ý
```suggestion
        Route::get('{user}'/tasks, 'showUserTasks')->name('users.tasks');
```
Thế thì e phải để ntn vì route sẽ có dạng :  users/1/tasks là task của user có id = 1, còn để như ban đầu là users/tasks/1 thì sẽ bị nhầm tasks có id là 1

Reply:
zạ